src/cadenza.js

```diff
@@ -555,6 +555,7 @@ export class CadenzaClient {
    * @param {boolean} [options.useMapSrs] - Whether the created geometry should use the map's SRS (otherwise EPSG:4326 will be used)
    * @param {OperationMode} [options.operationMode] - The mode in which a workbook should be operated
    * @param {AbortSignal} [options.signal] - A signal to abort the iframe loading
+   * @param {Object[]} [options.additionalLayers] - Layer definitions to be imported and shown in the background, as a basis for the drawing. Layer definition, with name, type and content (a Geojson featureCollection).
```

Comment:
(Needs resolution)
Please define a proper `LayerDefinition` type or - if it's feature collections - use the existing `FeatureCollection` type. `Object` is not helpful.

This would also make the textual description of this type here obsolete.